decidim-proposals/app/views/decidim/proposals/proposals/_vote_button.html.erb

```diff
@@ -1,27 +1,25 @@
-<% unless proposal.rejected? %>
-  <div id="proposal-<%= proposal.id %>-vote-button">
-    <% if !current_user %>
-      <% if current_settings.votes_blocked? %>
-        <%= action_authorized_button_to :vote, t('.votes_blocked'), proposal_proposal_vote_path(proposal_id: proposal, from_proposals_list: from_proposals_list), class: "button #{vote_button_classes(from_proposals_list)} disabled", disabled: true %>
-      <% else %>
-        <%= action_authorized_button_to :vote, t('.vote'), proposal_proposal_vote_path(proposal_id: proposal, from_proposals_list: from_proposals_list), class: "button #{vote_button_classes(from_proposals_list)}", data: { disable: true, "redirect-url": proposal_path(proposal) } %>
-      <% end %>
+<div id="proposal-<%= proposal.id %>-vote-button">
```

Comment:
I just removed the first and last line : 

`<% unless proposal.rejected? %>` 
`<%end%>` 

and moved it into `_proposal_list.html.erb `and `_proposal_column.html.erb`